app/code/Magento/LoginAsCustomerAssistance/Model/IsCustomerEnabled.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\LoginAsCustomerAssistance\Model;
+
+use Magento\Framework\App\ResourceConnection;
+
+/**
+ * Check if customer is Enabled
+ */
+class IsCustomerEnabled
```

Comment:
Hello @SilinMykola thank you very much for your contribution!

Service classes like this one should not use the connection directly. 

They should use a resource model class that, in turn, uses the connection.

This is the common convention in Magento that prevents mixing up abstraction layers.

You can refer to the `\Magento\LoginAsCustomerAssistance\Model\IsAssistanceEnabled` as an example of service class that uses `\Magento\LoginAsCustomerAssistance\Model\ResourceModel\GetLoginAsCustomerAssistanceAllowed` as a resource model.

Let me know if you need more details.

My best,
Alessandro

Reply:
Hello @aleron75 ! Thanks for your review and suggestions! I made changes, can you check, please?

Reply:
Hello @SilinMykola, good job!